prawns/tips/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "squid-tips",
+  "description": "Squid processors for substrate tips.",
+  "version": "0.0.1",
+  "private": true,
+  "scripts": {
+    "db:init": "docker-compose -f ../../docker-compose.dev.yml down && docker-compose -f ../../docker-compose.dev.yml up -d && yarn build && yarn migrate",
+    "dev:khala": "node -r dotenv/config lib/processors/khalaProcessor.js",
+    "dev:polkadot": "node -r dotenv/config lib/processors/polkadotProcessor.js",
+    "dev:kusama": "node -r dotenv/config lib/processors/kusamaProcessor.js",
+    "process:khala": "node lib/processors/khalaProcessor.js",
+    "process:polkadot": "node lib/processors/polkadotProcessor.js",
+    "process:kusama": "node lib/processors/kusamaProcessor.js",
+    "query-node": "squid-graphql-server",
+    "build": "rm -rf lib && tsc",
+    "codegen": "sqd codegen",
+    "migrate": "sqd db:migrate",
+    "migrate:revert": "sqd db revert",
+    "migration": "sqd db:create-migration Data"
+  },
+  "dependencies": {
+    "@polkadot/api": "^7.13.1",
+    "@polkadot/util-crypto": "^8.6.1",
+    "@subsquid/graphql-server": "^0.1.5",
+    "@subsquid/ss58": "^0.0.4",
+    "@subsquid/substrate-processor": "^0.4.1",
+    "dotenv": "^10.0.0",
+    "pg": "^8.7.1",
+    "typeorm": "^0.2.41"
+  },
+  "devDependencies": {
+    "@subsquid/cli": "^0.1.4",
+    "@types/node": "^16.11.17",
+    "@typescript-eslint/eslint-plugin": "^5.10.0",
+    "eslint": "7.26.0",
+    "prettier": "^2.3.2",
+    "typescript": "~4.5.4"
+  },
+  "resolutions": {
```

Comment:
do we need this? is it safer/easier to use fixed versions? I don't know much about resolutions so apologies if I'm missing the point - I'm assuming we needed this in litentry graph as it is a monorepo

Reply:
Me and @franjs added this so we get the proper version we need to have on the sub-dependencies. 
Some of the packages we are using might not get updates to the sub-dependencies and by adding this we guarantee we have the version of the package we need.

These resolutions in particular added some types that we needed when calling the API to get the `deposit`.